.github/workflows/docker.yml

```
@@ -0,0 +1,59 @@
+name: Docker Publish
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - develop
+    tags:
+      - release/*
```

```suggestion
      - v*.*.*
```

ref: https://github.com/ckb-cell/rgbpp-sdk/releases